clients/osv/src/main/kotlin/OsvApiClient.kt

```diff
@@ -44,24 +44,41 @@ import retrofit2.http.Path
  */
 interface OsvApiClient {
     companion object {
-        const val SERVER_URL_PRODUCTION = "https://api.osv.dev"
-        const val SERVER_URL_STAGING = "https://api-staging.osv.dev"
         const val BATCH_REQUEST_MAX_SIZE = 1000
 
         val JSON = Json.Default
 
+        /**
```

Comment:
commit-message-nit: "Mutual exclusive use" seems a bit confusing to me. It was obviously mutually exclusive before, because there is just on single parameter of type string which means: only one value can be passed, not multiple.

Reply:
I agree and have dropped that part from the commit message.